modulemd/v2/modulemd-profile.c

```diff
@@ -46,6 +46,17 @@ enum
 static GParamSpec *properties[N_PROPS];
 
 
+gboolean
+modulemd_profile_equals_wrapper (const void *a, const void *b)
+{
+  g_return_val_if_fail (MODULEMD_IS_PROFILE ((ModulemdProfile *)a), FALSE);
+  g_return_val_if_fail (MODULEMD_IS_PROFILE ((ModulemdProfile *)b), FALSE);
+
+  return modulemd_profile_equals (MODULEMD_PROFILE ((ModulemdProfile *)a),
```

Comment:
Same as above; you don't need to double-cast here.
```suggestion
  return modulemd_profile_equals ((ModulemdProfile *)a,
```